controllers/configuration_controller.go

```diff
@@ -44,7 +44,7 @@ import (
 const (
 	terraformInitContainerImg = "busybox:latest"
 	// TerraformImage is the Terraform image which can run `terraform init/plan/apply`
-	terraformImage     = "oamdev/docker-terraform:1.0.5"
```

Comment:
I have figure out why 1.0.5 doesn't work https://github.com/oam-dev/terraform-controller/pull/95. Let's roll back It right now.